cmake/std/PullRequestLinuxGCC7.2.0TestingSettings.cmake

```diff
@@ -20,6 +20,10 @@ set (Trilinos_ENABLE_COMPLEX_DOUBLE ON CACHE BOOL "Set by default for PR testing
 # Disable just one Teko sub-unit test that fails with openmpi 1.10 (#2712)
 set (Teko_DISABLE_LSCSTABALIZED_TPETRA_ALPAH_INV_D ON CACHE BOOL "Temporarily disabled in PR testing")
 
+# Disable the test of Modules in python for python 3 - this build only.
+# re-enable when python 3 and modules decide to play nice (likley RHEL8)
+set (TrilinosFrameworkTests_Modules_UnitTests_DISABLE ON CACHE BOOL "Temporarily disabled in PR testing")
```

Comment:
@prwolfe can we get these tests working before merging this PR?  It would be nice to protect this stuff in PR testing if possible.

Reply:
The Teko test has been disabled for some time. The Modules.py test will not run under python3 until the modules package fixes their python interface.  I could mock that out if it would import, but it won't.

I have a similar issue with your python usage in the get_changed_trilinos_packages, but that is due to an issue with the sems installation - I have a ticket open with them, Do you expect the rest of your code to pass testing under python3 or is this an unknown quantity?

Reply:
@prwolfe, we can fix the code to run with python 3 as well.  Why is python 2 being used in some PR builds and python 3 being used in others?

Reply:
We are trying to get ahead of this issue - python 3 will be the default in RHEL 8 so I was attempting to add tests in 3. Not urgent by any means.